.gitpod.yml

```diff
@@ -1,5 +1,23 @@
 tasks:
-  - command: ./main.sh
+  - name: Setup environment variables
+    command: |
+      source ./setup-env-variables.sh
+      gp sync-done env-setup
+  - name: Install AWS CLI
+    command: |
+      gp sync-await env-setup
+      ./install-aws-cli.sh
+      gp sync-done aws-cli
```

Comment:
**Question:** How come you added the sync + async commands here rather than running all of them in sequence? To get separate log output for each logged in tool?